src/main/resources/application.yml

```diff
@@ -98,3 +98,11 @@ sandbox:
         indexed:
           queue: sandbox.record.indexed
           dlq: ${sandbox.rabbitmq.queues.record.indexed.queue}.dlq
+
+predefinedSchemas: EDM-INTERNAL,EDM-EXTERNAL
+predefinedSchemas.EDM-INTERNAL.url:
+predefinedSchemas.EDM-INTERNAL.rootLocation:
+predefinedSchemas.EDM-INTERNAL.schematronLocation:
+predefinedSchemas.EDM-EXTERNAL.url:
+predefinedSchemas.EDM-EXTERNAL.rootLocation:
+predefinedSchemas.EDM-EXTERNAL.schematronLocation:
```

Comment:
In true YAML fashion, could this be reworded as follows?
```
predefinedSchemas: EDM-INTERNAL,EDM-EXTERNAL
  EDM-INTERNAL:
    url:
    rootLocation:
    schematronLocation:
  EDM-EXTERNAL:
    url:
    rootLocation:
    schematronLocation:
```
Or would that not work?

Reply:
Also: From the point of view of the configuration file, would it not make more sense to include it in the validation specific section at sandbox.validation? More configuration will/may be needed for other components, and perhaps it would be good to structure this properly. Even if this has some effect in the code, this may be worth it. What do you think?

Reply:
Yes I actually try that, but it does not work because of the nature of the properties
```
predefinedSchemas: EDM-INTERNAL,EDM-EXTERNAL
  EDM-INTERNAL:
```
You cannot do a property with a value and then a hierarchy, YAML will not work, I look around and this one was the way to go when having this kind of hierarchy. On the other hand I could reorder to something like
```
sandbox:
  validation:
    predefinedSchemas: 
      schemas: EDM-INTERNAL,EDM-EXTERNAL
      EDM-INTERNAL:
        url:
        rootLocation:
        schematronLocation:
      EDM-EXTERNAL:
        url:
        rootLocation:
        schematronLocation:
```
And then just adjust it when using the method I need.
I will make a note for MET-2541 to look into this

Reply:
[MET-2541](https://europeana.atlassian.net/secure/RapidBoard.jspa?rapidView=61&modal=detail&selectedIssue=MET-2541)

Reply:
Yes, I was afraid of that... your suggested structure looks like an improvement. We may not actually need the 'schemas' line: it is implied by the two children that its parent has.